core/src/main/scala/cr/pulsar/Client.scala

```diff
@@ -19,7 +19,7 @@ package cr.pulsar
 import cats.effect.{ Resource, Sync }
 import org.apache.pulsar.client.api.{ PulsarClient => Underlying }
 
-object PulsarClient {
+object Client {
```

Comment:
How about calling it `Pulsar` instead? `Client` seems too generic and we will more likely have conflicts (e.g. http4s' `Client`).

Reply:
That makes sense. Also, I was thinking about making it possible to create producer/consumer from `Client` so that we don't need to pass it around. However, this needs to be rethinked